thirdparty/oidn/common/platform.h

```diff
@@ -18,7 +18,9 @@
 
 #if defined(_WIN32)
   #define WIN32_LEAN_AND_MEAN
-  #define NOMINMAX
+  #ifndef NOMINMAX
+    #define NOMINMAX
+  #endif
```

Comment:
All changes in the thridparty code should be marked with `// -- GODOT start --` and `// -- GODOT end --` comments and listed in the `README.md`.